dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/TrustedHeaderOriginLists.java

```diff
@@ -21,23 +21,23 @@
 
 import com.ibm.websphere.ras.Tr;
 import com.ibm.websphere.ras.TraceComponent;
-import com.ibm.ws.http.channel.internal.filter.FilterList;
-import com.ibm.ws.http.channel.internal.filter.FilterListFastStr;
-import com.ibm.ws.http.channel.internal.filter.FilterListSlowStr;
-import com.ibm.ws.http.channel.internal.filter.FilterListStr;
+import com.ibm.ws.tcpchannel.internal.FilterList;
+import com.ibm.ws.tcpchannel.internal.FilterListFastStr;
+import com.ibm.ws.tcpchannel.internal.FilterListSlowStr;
 
 /**
  * Parses and keeps track of the HttpDispatcher trustedHeaderOrigin and trustedSensitiveHeaderOrigin configurations
  * Adapted from com.ibm.ws.tcpchannel.internal.AccessLists
  */
+/* 18752 */
 public class TrustedHeaderOriginLists {
 
     private static final TraceComponent tc = Tr.register(TrustedHeaderOriginLists.class);
 
-    private FilterList trustedAddresses = null;
+    private com.ibm.ws.tcpchannel.internal.FilterList trustedAddresses = null;
```

Comment:
we should be able to omit the full package now

Reply:
Not sure when/why Eclipse decided to do this but gone now.